tuning-fork/boilerplate
# 328/export as pdf docx

## 🤺 Summary


## 📝 Checklist

* [x] Are there any deployment considerations?
  * [x] Environment variables?
  * [x] External dependencies? (apis, dbs, infrastructure)
  * [x] Data or schema migrations?

## 🧶 Steps to run
Run npm i to add new packages.

## 🔬 Steps to test
1. Navigate to a grant show page.
2. Select the 'Export' on the right side of the grant show hero section.
3. A modal will appear with options to export the grant.
4. Select 'Export As Docx' and check that you have downloaded a grant.
5. Confirm that you can open the grant in a word processor program and on google docs.
